sam-styles/packages/components/button/styles/button.scss

```diff
@@ -831,4 +831,57 @@ $button-shadow: 0 0.25rem 0.5rem 0 rgba(0, 0, 0, 0.2);
     @include u-flex("column");
     flex-grow: 1;
   }
-}
\ No newline at end of file
+}
+
+.usa-button-pill {
+  @include u-radius("pill");
+  @include u-display("flex");
+  @include u-flex('align-center'); // Vertically align items in the center
+  @include u-flex('justify-center'); // Horizontally center content
+
+  svg {
+    width: 16px !important;
```

Comment:
Does important really need to be on everything to work?

Reply:
actually we do not need it, I was trying something else before so I added it but, since we do not need it, I will remove.